backend/app/core/utils.py

```
@@ -58,5 +90,38 @@ def format_datetime(dt: Optional[datetime]) -> Optional[str]:
+        return None
+
+async def get_users_with_permission(permission: str, db: AsyncSession) -> List[Users]:
+    """Get all users who have a specific permission through their roles."""
+    try:
+        query = select(Users).join(
+            UserRoles, UserRoles.user_id == Users.user_id
+        ).join(
+            Roles, Roles.role_id == UserRoles.role_id
+        ).where(
+            and_(
+                Users.is_active == True,
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `Users.is_active:` for truth checks

Replace with `Users.is_active`

(E712)